HLTrigger/btau/src/HLTmumutktkFilter.cc

```diff
@@ -134,17 +134,17 @@ bool HLTmumutktkFilter::hltFilter(edm::Event& iEvent, const edm::EventSetup& iSe
         if (tkRef == iVec) {mucandVec.push_back(cand); break;}
       }
     }
-    if(mucandVec.size()!= 2) throw cms::Exception("BadLogic") << "HLTmumutktkFilterr: ERROR: the vertex must have "
-                                                              << " exactly two muons by definition."  << std::endl;
+    if(mucandVec.size() < 2) throw cms::Exception("BadLogic") << "HLTmumutktkFilterr: ERROR: the vertex must have "
```

Comment:
thinking more - why is this an exception and not just a skip ahead to consider the next vertex?

Reply:
This is a sanity check in the logic of the path:
since, in the producer, the vertex is built with two muons and two tracks from the same collections used here, there would be some problem if they cannot be found anymore